### DISCUSSION TITLE
Stand alone installation?

### DISCUSSION THREAD

**Discussion by vfilter** [redpanda-data/connect#2712] [General]
Are there any plans to offer a standalone installation without all the added Redpanda baggage? Installing things with brew have increased package size from 22mb to 120mb with added complexity on the command line without any tangible benefit. We really liked Benthos and are sad about the sudden transition with degradation in documentation quality and ease of use. Would be great to know whether to go all in on Bento or stick with this.🤞

**Comment by mihaitodor** [comment]
Hey @vfilter 👋 The standalone GitHub releases which contain prebuilt artifacts for all supported platforms will always be available [here](https://github.com/redpanda-data/connect/releases). This is just the `redpanda-connect` (formerly `benthos`) static binary produced by `go build` which doesn't include the `rpk` tool. You can download the archive, decompress it and set it up as you wish.

Alternatively, if you have Go installed, you can `go install` it like so:
```
> git clone https://github.com/redpanda-data/connect.git
> cd connect
> git checkout tags/v4.30.1 # or whichever is the latest tag
> go install ./cmd/redpanda-connect
```

Unfortunately, it's not currently possible to run `go install github.com/redpanda-data/connect/v4/cmd/redpanda-connect@latest` due to some limitations of Go modules: https://github.com/golang/go/issues/44840.

We used to have support for https://asdf-vm.com as well (see [here](https://github.com/benthosdev/benthos-asdf)), but someone needs to look into updating that. I don't have the bandwidth right now, but PRs are welcome.

Note: We try to prevent the prebuilt all-in-one binary from blowing up in size, but many connectors rely on 3rd party libraries which we don't have any control over. The only option is to not include them by default if they cross certain limits. In practice, users should consider crafting their own binary and importing selectively [whichever components](https://github.com/redpanda-data/connect/tree/main/public/components) they need as described here: https://github.com/redpanda-data/redpanda-connect-plugin-example.

> degradation in documentation quality and ease of use. 

Would you mind providing some extra details on this? There is ongoing effort to improve things, but we need actionable feedback to share with the documentation team.

**Comment by Jeffail** [comment]
There's also going to be standalone docker images pushed to https://hub.docker.com/r/redpandadata/connect, this will start at the next tagged released.

**Reply by tbnguyen1407** [reply]
Hello, for the new artifact suffix "-community" and "-cloud", how will it affect the docker image tags? I don't see extra tags for those on dockerhub. Thanks
